src/org/etools/j1939_84/controllers/part06/Part06Step02Controller.java

```diff
@@ -54,9 +59,36 @@
     @Override
     protected void run() throws Throwable {
         // 6.6.2.1.a Global DM5 [(send Request (PGN 59904) for PGN 65230 (SPNs 1218-1219)]).
+        RequestResult<DM5DiagnosticReadinessPacket> globalDM5Result = getDiagnosticMessageModule().requestDM5(getListener());
```

Comment:
nit: This line and the next could be combined.  There's no need for the `globalDM5Result` variable.